src/main/resources/alfresco/messages/schema-update.properties

```diff
@@ -16,6 +16,7 @@ schema.update.warn.dialect_substituting=The dialect ''{0}'' is being changed to
 schema.update.err.forced_stop=The property 'stopAfterSchemaBootstrap' has been set.  The bootstrap process is being terminated.
 schema.update.err.found_multiple=\nMore than one Alfresco schema was found when querying the database metadata.\n   Limit the database user's permissions or set the 'hibernate.default_schema' property in 'custom-hibernate-dialect.properties'.
 schema.update.err.previous_failed=A previous schema upgrade failed or was not completed.  Revert to the original database before attempting the upgrade again.
+schema.update.err.upgrade_in_progress_on_another_node=A long running upgrade was found to be running on another node of the cluster. This node is shuting down.

```

Comment:
shutting

Reply:
Please check the l10n.properties and, if this file is localised, please raise a LOC issue.